### ISSUE TITLE
running in to "error: attribute 'lib' missing" while trying to nix-build release0.nix

### ISSUE THREAD

**Issue by CatspersCoffee** [Gabriel439/haskell-nix#95]
Following instructions on Project0, 

```$ nix-build release0.nix```

then get hit with:

```
$ nix-build release0.nix
error: attribute 'lib' missing

       at /home/user/haskell-nix/project0/project0.nix:9:13:

            8|   executableHaskellDepends = [ base ];
            9|   license = stdenv.lib.licenses.bsd3;
             |             ^
           10| }
(use '--show-trace' to show detailed location information)
```

Could anyone suggest a solution?

could it be that "stdenv.lib has recently been removed (in favor of just lib)" ?

**Comment by rcannon** [comment]
Running `cabal2nix . > project0.nix` before `nix-build release0.nix` fixes the problem for me. From what I can tell, it does just replace `stdenv.lib` with just `lib`.

**Comment by Gabriel439** [comment]
Fixed by 2c624fa680ef16150195066e891ac3f54d27d558
